src/components/Editor/reducer.js

```diff
@@ -60,11 +65,15 @@ const editor = (state = defaultEditorState, action) => {
                 ]
             }
         case 'DELETE_MODULE':
+            let modules = [...state.modules]
```

Comment:
I'm just creating a duplicate of modules, deleting the module we were asked to (from the duplicate), then focusing the prior module.  It's not the cleanest, but it works for now and can be cleaned up in a later version.  It was the simplest solution I could find to do what I was wanting.

If you know of a better way, please, feel free to commit to this PR or create a new PR because I, honestly, don't know the best way to accomplish this task.